libs/remix-debug/test/sourceLocationTracker.js

```diff
@@ -63,12 +63,12 @@ tape('SourceLocationTracker', function (t) {
         st.equal(map.start, 35)
 
         map = await sourceLocationTracker.getSourceLocationFromVMTraceIndex('0x0d3a18d64dfe4f927832ab58d6451cecc4e517c5', 45, output.contracts)
-        st.equal(map.file, -1)
```

Comment:
ah I understand the question earlier now.
so this is not that the src map has changed, but the bytecode and thus the vm trace has changed

Reply:
ok, got it.